### PR TITLE
Upgrade from Django 2.0 to Django 3.0

### PR DESCRIPTION
**Pull request by UsamaSadiq**
### Description
- Subsequent PR after https://github.com/openedx/openedx-events/pull/104 to Upgrade the package from `Django 2.0` to `Django 3.0`.

### CODE REVIEW

**Comment by mariajgrimaldi** [comment]
Hello there! I'm tagging you here, so you're aware of this change @timmc-edx @robrap @felipemontoya. We're currently testing against an old Django version, and this PR updates it according to the constraints of edx-platform@master.

**Comment by robrap** [comment]
Ok. Thank you.

File: tox.ini [hunk 1]
```diff
@@ -40,7 +40,7 @@ norecursedirs = .* docs requirements site-packages
 deps =
     -r{toxinidir}/requirements/test.txt
 commands =
-    pytest {posargs}
```
Comment (mariajgrimaldi) [inline tox.ini:43]: what about this change? :thinking:

Comment (UsamaSadiq) [reply tox.ini:43]: Explicitly mentioning the target directories is a standard practice being followed across edX/OpenedX repos so I updated it accordingly.

Comment (mariajgrimaldi) [reply tox.ini:43]: Good! Thanks!

**Review by mariajgrimaldi — APPROVED** [review]

**Pull request merged**